source/state_representation/include/state_representation/space/cartesian/CartesianPose.hpp

```diff
@@ -291,23 +291,25 @@ class CartesianPose : public CartesianState {
   friend CartesianPose operator*(double lambda, const CartesianPose& pose);
 
   /**
-   * @brief Overload the /= operator with a scalar
-   * @param lambda The scalar to divide with
-   * @return The Cartesian pose divided by lambda
+   * @brief Scale inplace by a scalar
+   * @copydetails CartesianState::operator*=(double)
+   * @param lambda The scaling factor
+   * @return The reference to the scaled Cartesian pose
    */
   CartesianPose& operator/=(double lambda);
 
   /**
-   * @brief Overload the / operator with a scalar
-   * @param lambda The scalar to divide with
-   * @return the Cartesian pose divided by lambda
+   * @brief Scale a Cartesian pose by a scalar
+   * @copydetails CartesianState::operator*=(double)
+   * @param lambda The scaling factor
+   * @return The scaled Cartesian acceleration
    */
   CartesianPose operator/(double lambda) const;
 
   /**
-   * @brief Overload the / operator with a time period
-   * @param dt The time period to divide by
-   * @return The corresponding Cartesian twist
+   * @brief Derive a Cartesian pose over a time period
```

Comment:
nitpick: `Differentiate` would be more appropriate than `Derive` in my opinion. "Derive the pose" to me implies "calculating / solving" so that you end up with a pose. The verb "differentiate" is more clear, or alternatively "take the derivative"